R/robustbayesianmetaanalysis.R

```diff
@@ -811,11 +815,11 @@ RobustBayesianMetaAnalysis <- function(jaspResults, dataset, options, state = NU
        length(priors[["tau_null"]]) == 0) ||
       (length(priors[["omega"]]) == 0 &&
        length(priors[["omega_null"]]) == 0)) {
-    priors_error <- createJaspTable()
-    priors_error$setError(gettext(
+    priorsError <- createJaspTable()
+    priorsError$setError(gettext(
         "At least one prior distribution per parameter must be specified (either null or alternative)."
```

Comment:
I'm not sure what this error message means. What confuses me is the the "either null or alternative" part.

Reply:
The analysis requires you to specify priors for the Null and the Alternative hypothesis for each parameter. For the analysis to run and estimate the ensemble, at least one of them for each parameter needs to be specified

Reply:
Perhaps it's better to translate this to actions that the user can do to resolve the problem. You might want to consider adding a sentence such as "Please specify a prior distribution for each parameter in the Model specification section" or something like that.

Reply:
How do we do other warnings in JASP? We need to adopt a consistent style.